subgraphs/accounts/data.js

```diff
@@ -12,3 +12,22 @@ export const ACCOUNTS = [
     tier: "PREMIUM"
   },
 ];
+
+export const TRANSACTIONS = [
```

Comment:
We don't actually need to duplicate this data here. The transactions subgraph already takes care of adding transactions to bank accounts and links to bank accounts from transactions

https://github.com/apollosolutions/financial-supergraph/blob/134b6dfcc229c78911d8503fca396b4fd04faf4a/subgraphs/transactions/schema.graphql#L8-L19